appveyor.yml

```diff
@@ -24,7 +24,6 @@ matrix:
   fast_finish: true
 install:
 - SET PATH=C:\Ruby%RUBY_VER%\bin;%PATH%
-- gem install bundler --quiet --no-ri --no-rdoc
```

Comment:
Yes stable first. Current thinking is I may not be able to merge this stuff up to master.  Instead, fresh PRs for master